src/app/members/components/MemeberHeader.tsx

```diff
@@ -59,7 +59,7 @@ export const MemeberHeader = () => {
                 />
                 <MagnifyingGlassIcon className="absolute left-3 top-1/2 transform -translate-y-1/2 text-gray-600 w-4 h-4" />
               </div>
-              <Button data-testid="addMemberButton" className="h-7">
+              <Button data-testid="addMemberButton" className="h-7"  onClick={()=> router.push(Routes.AddMember)}>
```

Comment:
How about we just wrap this up in Next.js's Link component?

Reply:
Fixed